pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -91,6 +91,10 @@ func (c *Client) Setup(t *testing.T) *Client {
 	return c
 }
 
+func (c *Client) Close() {
+	c.server.Close()
```

Comment:
> add nil check

`httptest.Server.Close()` does not return error.

> clear so multiple calls to Close() are safe regardless of state.

Read this a few times, but can't make out what you mean.